doc/intro.rst

```diff
@@ -59,9 +59,10 @@ and verifying that no error message appears.  It may be necessary to install
 from the `slycot project page <https://github.com/python-control/Slycot>`_.
 
 For users with a working the Anaconda distribution of Python, the following
```

Comment:
Should this be "...with the Anaconda distribution..."?

(copied from <https://github.com/python-control/python-control/pull/123#discussion_r94277363>; still relevant here)